### PR TITLE
Update api_model.cpp

### PR DESCRIPTION
**Pull request by lshems**
Added carryTrim to input API's.

First pull request ever, so check carefully :).

Hope I did all as explained in the developers wiki.

### CODE REVIEW

**Comment by bsongis** [comment]
Will fix #6491

**Review by bsongis — APPROVED** [review]
Seems OK to me.

File: radio/src/lua/api_model.cpp [hunk 1]
```diff
@@ -413,6 +415,9 @@ static int luaModelInsertInput(lua_State *L)
       else if (!strcmp(key, "curveValue")) {
         expo->curve.value = luaL_checkinteger(L, -1);
       }
+      else if (!strcmp(key, "carryTrim")) {
+        expo->carryTrim= lua_toboolean(L, -1);
```
Comment (3djc) [inline radio/src/lua/api_model.cpp:419]: space before "="

Comment (bsongis) [reply radio/src/lua/api_model.cpp:419]: right, I didn't see it.
cf our coding rules

**Comment by lshems** [comment]
Ok. Thanks.

I won't be creating much new stuff, but I can complete, add and improve some of the Lua API's.

Hope it is appreciated.

**Pull request merged**